development/language/validation.rst

```diff
@@ -271,4 +271,4 @@ License
 * All translations must be released under
   `GNU General Public License 2.0 <http://www.opensource.org/licenses/gpl-2.0.php>`_
 
-.. _Customisation Database: https://www.phpbb.com/go/customise/language-packs
+.. _Customisations Database: https://www.phpbb.com/customise/db/language_packs-25
```

Comment:
With the actual redirect, yes. But I think it is confusing, if we would use the `/3.1` link in 3.2.x and master/3.3 documentation, too. Although this will help us to stay flexible, if we want to change this behaviour in the future.